release/ray_release/cluster_manager/minimal.py

```diff
@@ -20,6 +21,8 @@ class MinimalClusterManager(ClusterManager):
     Builds app config and compute template but does not start or stop session.
     """
 
+    extra_tags_resource_types = ["instance", "volume", "host-reservation"]
```

Comment:
What are these? 

Also it seems we don't override this ever, so can we just have a constant instead?